wiki/Wiki.md

```diff
@@ -80,3 +80,15 @@ Our wiki consists of the following components:
 If you change the wiki through the web interface at https://github.com/oppia/oppia-android/wiki, the [`wiki.yml`](https://github.com/oppia/oppia-android/blob/develop/.github/workflows/wiki.yml) workflow will be triggered and the wiki will be reset to by pushing the files from the `/wiki` folder to the `oppia/oppia-android.wiki` git repository.
 > **Note**  
 > The `/wiki` folder in the `oppia/oppia-android` repository is the single source of truth for our wiki.
+
+## Tips and Tricks
+
+* ### How can I see markdown (wiki) preview in Android Studio?
+
+1. Start the IDE, use Help -> Find Action, type `Choose Boot Java Runtime for the IDE`, press Enter.
+2. Select the newest version with JCEF to install.
```

Comment:
This process includes ignoring a warning that explicitly instructs to **not** change the Java runtime--is this actually necessary? What of upgrading Java actually lets Markdown previewing work?

Reply:
Yes, @BenHenning, It's giving a warning, I will find another way to get a preview.

Reply:
Talked over chat--a TODO is filed & mentioned to track updating the wiki once this step is no longer needed.